src/jit/flowgraph.cpp

```diff
@@ -9000,7 +9000,7 @@ void Compiler::fgSimpleLowering()
                         add->gtNext  = tree;
                         tree->gtPrev = add;
 #else
-                        range.InsertAfter(arr, con, add);
+                            range.InsertAfter(arr, con, add);
```

Comment:
For the life of me, I cannot figure out why clang-format keeps wanting to indent this. It drives me nuts, and I've seen it on multiple of my own format runs, and I know there's nothing we can do about it right now, but it's just strange.